src/rules/no-deprecated-flash.js

```diff
@@ -0,0 +1,132 @@
+'use strict'
+
+const {getJSXOpeningElementName} = require('../utils/get-jsx-opening-element-name')
+const {isPrimerComponent} = require('../utils/is-primer-component')
+const url = require('../url')
+
+/**
+ * @type {import('eslint').Rule.RuleModule}
+ */
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Flash component is deprecated. Use Banner from @primer/react/experimental instead.',
+      recommended: true,
+      url: url(module),
+    },
+    fixable: 'code',
```

Comment:
I don't think this issue is always fixable so just throw a warning in lint for now and remove all the logic to fix this lint error